client/my-sites/earn/memberships/index.jsx

```diff
@@ -301,6 +286,36 @@ class MembershipsSection extends Component {
 		);
 	}
 
+	getWording( subscriber ) {
```

Comment:
Super nitpicky and non-blocking. This function only returns a portion of the wording needed by the component. On an ideal scenario, I think this function should return all the needed messages for consistency. Otherwise, developers might not know when to use this function.

Reply:
I think that would be more consistent, but in general, moving the messages further away from their context makes things more difficult to read.

Would renaming this function to something like `getIntervalDependantWording` help?

Reply:
> Would renaming this function to something like `getIntervalDependantWording` help?

Yup, that'd solve my main concern of not knowing where translations should be defined.